src/storage/serialization/serialize_parse_info.cpp

```diff
@@ -482,4 +485,15 @@ unique_ptr<ParseInfo> VacuumInfo::Deserialize(Deserializer &deserializer) {
 	return std::move(result);
 }
 
+void AddConstraintInfo::Serialize(Serializer &serializer) const {
```

Comment:
These methods are generated by `scripts/generate_serialization.py` using `parse_info.json`

Reply:
I noticed too late :-). I will change that.

Reply:
Fixed.